relayer/src/worker/packet.rs

```diff
@@ -172,6 +169,9 @@ fn handle_packet_cmd<ChainA: ChainHandle, ChainB: ChainHandle>(
                 clear_interval,
                 height,
             ) {
+                // Decrement the height in order to avoid clearing packets at the same
+                // height at which the packet's events were queried
+                let height = height.decrement().unwrap();
```

Comment:
Is unwrapping here upon decrementing acceptable?

Reply:
I think we can do something like:
```
    match height.decrement() {
        Err(e) => Err(LinkError::decrement_height(height, e)),
        Ok(height) => link.a_to_b.schedule_packet_clearing(Some(height)),
    }
```